sktime/forecasting/model_selection/_tune.py

```diff
@@ -182,9 +182,21 @@ def _fit(self, y, X, fh):
         self : returns an instance of self.
         """
         cv = check_cv(self.cv)
-
-        scoring = check_scoring(self.scoring, obj=self)
-        scoring_name = f"test_{scoring.name}"
+        
+        if isinstance(self.scoring, list):
+            scoring = []
+            scoring_name = []
```

Comment:
Also please, if it is a list, calling it `scoring_name` is confusing i will expect a string here, because what kind of python Type a name can be otherwise?

If you call it `scoring_names` i know it is mroe than one, therefore i do probably expect a list.

Reply:
If I do change scoring_name to scoring_names here, I will have to modify the nomenclature further down where meta is defined. A simple resolution could be to use scoring_names for the non-list case as well

Reply:
ok we need to think of a good solution, but imo leaving a variable with a bad name because you have to refactor other code somewhere else is not a good idea. Instead it is a symptom that refactoring of the code is needed due to new features being included.

Reply:
Hmm, considering this, I think it'd be better to refactor the code altogether, as even this would not involve much of an overhaul and will distinguish scoring_name and scoring_names for clarity

Reply:
sounds def good to me :)